clients/admin-ui/src/features/plus/plus.slice.ts

```diff
@@ -262,8 +262,16 @@ const plusApi = baseApi.injectEndpoints({
         url: `plus/custom-metadata/custom-field-definition/resource-type/${resource_type}`,
       }),
       providesTags: ["Custom Field Definition"],
-      transformResponse: (list: CustomFieldDefinitionWithId[]) =>
-        list.sort((a, b) => (a.name ?? "").localeCompare(b.name ?? "")),
+      transformResponse: (
+        response: CustomFieldDefinitionWithId[] | { detail: string },
+      ) => {
+        if ("detail" in response) {
```

Comment:
I don't get the context for this change. Could you explain?

Reply:
That's a great question that I'm trying to investigate and remember...

Reply:
@lucanovera I updated this code with a comment to explain why. For some reason, instead of an error code or an empty list, the BE is returning { detail: "No custom metadata fields found with resource type system" } which was causing a console error in this original piece of code.

Reply:
Good idea! Thanks for doing that.